core/dataflow/src/main/resources/udf_operator.py

```diff
@@ -0,0 +1,112 @@
+from abc import ABC
+
+import pandas
+
+
+class UDFOperator(object):
```

Comment:
I suggest we add a prefix `texera` to the class name and the file name to avoid potential conflicts.

Reply:
OK.

Reply:
Done.